main.go

```diff
@@ -369,3 +372,45 @@ func Reload(logger *util.Logger) {
 	logger.PrintInfo("Successfully reloaded pganalyze collector (PID %d)\n", pid)
 	os.Exit(0)
 }
+
+// Delete any temp files we may have left behind on an unclean shutdown
+func pruneTempFiles(logger *util.Logger) {
+	usr, err := user.Current()
+	if err != nil {
+		logger.PrintWarning("Could not check current user to prune temp files: %s\n", err)
+		return
+	}
+	uid, err := strconv.ParseUint(usr.Uid, 10, 32)
+	if err != nil {
+		logger.PrintWarning("Could not parse current user uid to prune temp files: %s\n", err)
+		return
+	}
+	uid32 := uint32(uid)
+	tmpdir := os.TempDir()
+	entries, err := os.ReadDir(tmpdir)
+	if err != nil {
+		logger.PrintWarning("Could not open temp directory to prune temp files: %s\n", err)
+		return
+	}
+	for _, entry := range entries {
+		name := entry.Name()
+		if !strings.HasPrefix(name, "pga_collector") {
```

Comment:
Should we put "pga_collector" name (or possibly "pganalyze_collector") in a constant somewhere? (maybe move this utility + the constant into a file in `util/`, so it can be referenced from input/config?)